source/administration/monitoring/bucket-notifications.rst

```diff
@@ -211,6 +211,122 @@ You can only listen to these events through the `ListenNotification <https://min
 
 .. todo
 
+
+Payload schema
```

Comment:
I guess this is a descriptive enough heading?

Reply:
If you want to be extremely explicit, `Event Notification JSON Schema`.

Reply:
Used Ravind's longer recommendation